sources/packages/backend/eslint.config.ts

```diff
@@ -0,0 +1,29 @@
+import globals from "globals";
+import { defineConfig } from "eslint/config";
+import eslint from "@eslint/js";
+import tseslint from "typescript-eslint";
+import eslintConfigPrettier from "eslint-config-prettier";
+
+export default defineConfig(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  {
+    languageOptions: {
+      globals: {
+        ...globals.node,
+        ...globals.jest,
+      },
+    },
+    rules: {
+      "no-unsafe-optional-chaining": "warn",
+      "no-prototype-builtins": "warn",
+      "@typescript-eslint/no-explicit-any": "warn",
+      "@typescript-eslint/interface-name-prefix": "off",
```

Comment:
I could not see this rule anymore in https://typescript-eslint.io/rules/

Is it still a valid one?

Reply:
Removed.